content/pt-br/docs/setup/best-practices/multiple-zones.md

```diff
@@ -0,0 +1,79 @@
+{{< /note >}}
+
+## Comportamento do nó
+
+Kubernetes espalha automaticamente os Pods para
+recursos de carga de trabalho (tais como {{< glossary_tooltip text="Deployment" term_id="deployment" >}} ou {{{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}) através de diferentes nós em um cluster. Esta propagação ajuda
+reduzir o impacto das falhas.
+
+Quando os nós são iniciados, o kubelet em cada nó adiciona automaticamente {{< glossary_tooltip text="labels" term_id="label" >}} para o objeto Node que representa esse kubelet específico na API do Kubernetes. Esses rótulos podem incluir [informação sobre zonas](/docs/reference/labels-annotations-taints/#topologykubernetesiozone).
+
+Se seu cluster abrange múltiplas zonas ou regiões, você pode usar rótulos de nó em conjunção com [Restrições de propagação de topologia de pod](/docs/concepts/workloads/pods/pod-topology-spread-constraints/) para controlar como os pods são distribuídos em seu cluster entre os domínios de falha: regiões, zonas e até nós específicos.
+Essas dicas permitem que o {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} colocar Pods para melhor disponibilidade esperada, reduzindo o risco de uma falha afeta toda a sua carga de trabalho.
+
+Por exemplo, você pode definir uma restrição para garantir que as 3 réplicas de um StatefulSet estão sendo executadas em zonas diferentes para cada um, sempre que possível. Você pode definir isso declarativamente sem definir explicitamente quais zonas de disponibilidade estão em uso para cada carga de trabalho.
+
+### Distribuição de nós por zonas
+
+O núcleo do Kubernetes não cria nós para você; você precisa fazer isso você mesmo, ou utilizar uma ferramenta como o [Cluster API](https://cluster-api.sigs.k8s.io/) para
+gerir os nós para você.
+
+Utilizando ferramentas como o Cluster API, é possível definir conjuntos de máquinas para executar como nós de trabalho para seu cluster em vários domínios de falha e regras para recuperar automaticamente o cluster em caso de interrupção do serviço de toda a zona.
+
+## Atribuição manual de zonas para Pods
+
+Você pode aplicar [restrições de selecção de nó](/docs/concepções/concepções/cheduling-eviction/assign-pod-node-node/#nodeselector) aos pods que você cria, bem como aos modelos de pod em recursos de carga de trabalho como Deployment, StatefulSet ou Job.
+
+## Acesso de armazenamento para zonas
+
+Quando são criados volumes persistentes, `PersistentVolumeLabel`, o [controlador de admissão](/docs/reference/access-authn-authz/admission-controllers/) adiciona automaticamente etiquetas de zona a quaisquer Volumes Persistentes que estejam ligados a um
+zona. O {{{{< glossário_tooltip text="scheduler" term_id="kube-scheduler" >}} garante então, através do seu atributo `NoVolumeZoneConflict`, que os pods que reivindicam um determinados volumes persistentes sejam colocados apenas na mesma zona que esse volume.
```

Comment:
[Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/620ba2d6a204ae00076be834#L76).
This suggestion might resolve the error! 😊

```suggestion
zona. O {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} garante então, através do seu atributo `NoVolumeZoneConflict`, que os pods que reivindicam um determinados volumes persistentes sejam colocados apenas na mesma zona que esse volume.
```